lib/changeDirection.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-module.exports = function(newDirection) {
+module.exports = function (newDirection) {
 	this.context.clearRect(0, 0, this.x1, this.y1);
 	this.direction = newDirection;
 	this.validateInput('direction');
+	this.validateCustomDirection();
```

Comment:
Since there is no validation when you pass the value 'direction', decided to add a case to `validateInput` with the `direction` value. 
It will avoid to do the following:
```
this.validateInput('direction');
this.validateInput('customDirection');
```